qt/CMakeLists.txt

```diff
@@ -63,6 +63,11 @@ set(SRC
 
 omim_add_executable(${PROJECT_NAME} MACOSX_BUNDLE ${RES_SOURCES} ${SRC})
 
+target_sources(${PROJECT_NAME} PRIVATE
```

Comment:
Should this line be under `if(WIN32)` then?

Reply:
`if(PLATFORM_WIN)`

Reply:
On `!windows`, `.rc` files do not participate in builds are not even mentioned in the generated `build.ninja`.
`.manifest` files are mentioned in `build.ninja`, but they don't participate in build either.
I wrapped both in `if (PLATFORM_WIN)` and moved adding the manifest to `omim_add_executable`. Tests should use this manifest, too.